composer.json

```diff
@@ -6,33 +6,32 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.8",
+        "drupal/core": "^8.9",
         "ext-json": "*",
-        "php": ">=7.2"
+        "php": ">=7.3"
```

Comment:
It will break tests for all environments where PHP 7.2 is used. But officially Drupal 8 and PHP 7.2 aren't supported anymore.

Reply:
We don't support PHP 7.2 anymore.